packages/clay-tabs/src/List.tsx

```diff
@@ -114,6 +114,7 @@ export const List = React.forwardRef<HTMLUListElement, IProps>(function List(
 			className={classNames(
 				'nav nav-tabs',
 				{'nav-justified': justified},
+				{[`nav-tabs-${displayType}`]: displayType},
```

Comment:
I think it's better to just add when the type is `light` to make sure that previous versions don't add the type to `basic` or `underline`.

```suggestion
				{[`nav-tabs-${displayType}`]: displayType === 'light'},
```